.github/workflows/continuous-integration.yaml

```diff
@@ -40,8 +40,8 @@ jobs:
     name: Unit tests
     runs-on: ubuntu-latest
 
-    # max run time 5 minutes
-    timeout-minutes: 5
+    # max run time 10 minutes
+    timeout-minutes: 10
```

Comment:
Is this increase required because we are starting cassandra and coordinator for few Unit tests?

Reply:
Yes, its because of some of the new tests added.